src/core/qgssnappingconfig.h

```diff
@@ -168,18 +182,55 @@ class CORE_EXPORT QgsSnappingConfig
          */
         void setTypeFlag( QgsSnappingConfig::SnappingTypeFlag type );
 
-        //! Returns the tolerance
+        /**
+         * !
+         * Returns the tolerance
+         * \since QGIS 3.12
+         */
         double tolerance() const;
 
-        //! Sets the tolerance
+        /**
+         * Sets the tolerance
+         * \since QGIS 3.12
+         */
         void setTolerance( double tolerance );
 
-        //! Returns the type of units
+        /**
+         * Returns the type of units
+         * \since QGIS 3.12
+         */
         QgsTolerance::UnitType units() const;
 
-        //! Sets the type of units
+        /**
+         * Sets the type of units
+         * \since QGIS 3.12
+         */
         void setUnits( QgsTolerance::UnitType units );
 
+        /**
+         * Returns min scale on which snapping is limited
+         * \since QGIS 3.14
+         */
+        double minScale() const;
```

Comment:
both in the docstring and the method names can you use minimum and maximum in whole words rather than min/max?

Reply:
Done